src/app/services/raiden.service.ts

```diff
@@ -441,6 +441,34 @@ export class RaidenService {
             );
     }
 
+    public mintToken(
+        token: UserToken,
+        to: string,
+        value: number
```

Comment:
While these are the names of the query parameters, it would be better to change the name of the parameters on the method signature for clarity. I was thinking something like `target` and `amount` but if you have any better suggestion feel free to use it. It took me at least a couple of seconds to realize that `to` is the minting target.

Reply:
Good point.